code/script.js

```diff
@@ -225,101 +232,164 @@ const generateBoard = () => {
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+  console.log('secret person selected')
+  console.log(secret)
 }
 
 // This function to start (and restart) the game
 const start = () => {
+  console.log('start funktion is called!')
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
   // What else should happen when we start the game?
+  // set the secret person -> invoke set Secret funktion
+  setSecret()
+  //show the board on the screen -> invoking generateboard funktion. 
+  generateBoard()
 }
 
 // setting the currentQuestion object when you select something in the dropdown
 const selectQuestion = () => {
-  const category = questions.options[questions.selectedIndex].parentNode.label
-
+  const category = questions.options[questions.selectedIndex].parentNode.label  
+  const value = questions.options[questions.selectedIndex].value
   // This variable stores what option group (category) the question belongs to.
   // We also need a variable that stores the actual value of the question we've selected.
-  // const value =
-
+  
   currentQuestion = {
     category: category,
-    // value: value
+    value: value,
   }
+  console.log('question asked', currentQuestion)
 }
 
 // This function should be invoked when you click on 'Find Out' button.
+let count = 0
 const checkQuestion = () => {
-  const { category, value } = currentQuestion
+  count+= 1
+  guessCounter.innerHTML = " "
+  guessCounter.innerHTML += `<p> Guesses made: ${count}</p>`
```

Comment:
I recall you mentioning the counter as a stretch goal you had, nice work! However, the counter should be reset to 0 when the game is restarting.